marketplace_builder/views/pages/reference/form-configurations/validation.liquid

```diff
@@ -0,0 +1,246 @@
+slug: reference/form-configurations/validation
+searchable: true
+---
+
+This guide teaches you how to validate the state of objects before they go into the database using From Configuration's validations feature.
+
+After reading this guide, you will know:
+
+* How to configure form validation.
+* How to render validation errors.
+
+
+## Overview
+
+Here's an example of a very simple validation:
+
+<pre><code class="language-yaml">
+{% raw %}
+configuration:
+  properties:
+    title:
+      validation:
+        presence: true
+{% endraw %}
+</code></pre>
+
+As you can see, form with the above configuration will not be valid without title property passed with the request unless it's already set on the object passed to the form.
+
+
+## Validation Purpose
+
+Validations are used to ensure that only valid data is saved into your database.
+
+## Validation types
+
+You can use the follwing validation types:
+
+### presence
+<pre><code class="language-yaml">
+{% raw %}
+            title:
+              validation:
+                presence: true
+{% endraw %}
+</code></pre>
+### unique
+<pre><code class="language-yaml">
+{% raw %}
+     configuration:
+        properties:
+          slug:
+            validation:
+              presence: true
+              unique:
+                scope:
+                  - other_property
+                message: Must be uniq
+{% endraw %}
+</code></pre>
+### confirmation
+<pre><code class="language-yaml">
+{% raw %}
+    email:
+      validation:
+        confirmation: true
+{% endraw %}
+</code></pre>
+### inclustion / exclusion
+<pre><code class="language-yaml">
+   malpractice_history:
+      validation:
+        inclusion:
+          in: [true, false]
+          message: disclosure is required
+</code></pre>
+### format
+<pre><code class="language-yaml">
+    phone:
+      validation:
+        presence: true
+        format:
+          with: !ruby/regexp /\A[+]?[\d \-()]+\z/
+          allow_blank: true
+          message: 'This seems to be an invalid phone number. Please try using numbers only'
+</code></pre>
+### file_content_type
+<pre><code class="language-yaml">
+          file:
+            validation:
+              presence: true
+              file_content_type:
+                allow:
+                - image/jpeg
+                - image/png
+                - application/pdf
+                - application/msword
+                - application/vnd.openxmlformats-officedocument.wordprocessingml.document
+                - application/vnd.ms-excel
+                - application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+</code></pre>
+### length
+{% raw %}
+<pre><code class="language-yaml">
+    attachments:
+      validation:
+        length:
+          is: 2
+          message: "You need to include Front side and Back side Photo ID"
+</code></pre>
+{% endraw %}
+
+The possible length constraint options are:
+
+* :minimum - The attribute cannot have less than the specified length.
```

Comment:
I think that has been deprecated in favor of normal yaml keys - ruby symbols are not really intuitive for anyone other than ruby devs - correct me if im wrong @Slashek